mkdocs.yml

```diff
@@ -29,8 +29,8 @@ markdown_extensions:
         permalink: true
     - attr_list
     - pymdownx.emoji:
-        emoji_index: !!python/name:materialx.emoji.twemoji
-        emoji_generator: !!python/name:materialx.emoji.to_svg
+        emoji_generator: ""
+        emoji_index: ""
```

Comment:
Not sure if this change is needed. Is there a reason why it's modified?

Reply:
The netlify build fails with
```
12:03:37 PM: $ mkdocs build
12:03:37 PM: ERROR    -  Config value: 'markdown_extensions'. Error: unsupported callable
12:03:37 PM: Aborted with 1 Configuration Errors!
```

You can always test locally the docs site with `mkdocs serve`, from the root dir.

Reply:
ah thanks for that tip, I hadn't really looked beyond the example folder, so unsure what that is referring to but I will investigate.

Reply:
this does not error anymore but I did notice that the Title is lowercase as I didn't set it to uppercase. So can I change it manually?